### ISSUE TITLE
local-notification-utils.js has an error in it ?

### ISSUE THREAD

**Issue by Poketchu** [katzer/cordova-plugin-local-notifications#808]
Hi.

I get this issue that goes on my nerves since I didn't do anything else that adding the plugin to my project using the cordova CLI (cordova plugin add ...).
The plugin version i pulled is 0.8.3 and my situation is the following: the ionic.Platform.ready(function() { /* my code*/ }) is never called anymore which is a bummer since it handles all my db access and stuff.

Where the issue starts to be funky is that all I get using cordova serve or runing the installed apk on my Android phone are those lines in the console:
![image](https://cloud.githubusercontent.com/assets/9591559/12093970/432e0976-b306-11e5-9ff7-bde361a05085.png)

Once again, all I did was adding the plugin to my project and still didn't tried to use the code.
As I lurked in the plugin's code (in the local-notification-utils.js file), il found these lines:

```javascript
175    if (options.every)
176        if (device.platform == 'iOS' && typeof options.every != 'string') {
177            options.every = this.getDefaults().every;
178            var warning = 'Every option is not a string: ' + options.id;
179            warning += '. Expects one of: second, minute, hour, day, week, ';
180            warning += 'month, year on iOS.';
181            console.warn(warning);
182        }
183    }
```

It seems to me that there is a missing '{' at line 173 which makes the error occur at line 186. But after modifying the code "a la mano" (in the main plugin's folder and in those placed in the platforms\plugins folders) the issue still looks to be the same.

Could someone give me some help ?

**Comment by Poketchu** [comment]
Woops.

Just saw this '{ fixed' pushed yesterday stuff. Could someone explain me what version I shoud add using the CLI to get this last fix ?

**Comment by Poketchu** [comment]
Well...

Manualy fixed it... Can't wait for the merge to become CLI pullable.
The morals of this short strory: "don't git push it when it's broken or people will hate you". :)

See ya.

**Comment by katzer** [comment]
@Poketchu Recently released v0.8.4
